servers/rendering/renderer_rd/shaders/environment/sdfgi_preprocess.glsl

```diff
@@ -1033,7 +1033,7 @@ void main() {
 
 	if (local == ivec3(0) && store_position_count > 0) {
 		store_from_index = atomicAdd(dispatch_data.total_count, store_position_count);
-		uint group_count = (store_from_index + store_position_count - 1) / 64 + 1;
+		uint group_count = Math::division_round_up(store_from_index + store_position_count, 64);
```

Comment:
This should not have been merged, the shader does not even compile.
Was this PR even tested?

Reply:
My mistake, I added this yesterday.

Reply:
Fixed by 7abaac631bd897154686e6db030904d0af238248.